src/Orleans.Transactions/State/TransactionQueue.cs

```diff
@@ -286,20 +286,22 @@ public void NotifyOfPing(Guid transactionId, DateTime timeStamp, ParticipantId r
                 if (this.confirmationTasks.TryGetValue(transactionId, out var record))
                 {
                     // re-send now
-                    record.LastSent = null;
                     this.confirmationWorker.Notify();
                 }
                 else
                 {
                     // we never heard of this transaction - so it must have aborted
                     resource.Reference.AsReference<ITransactionalResourceExtension>()
-                            .Cancel(resource.Name, transactionId, timeStamp, TransactionalStatus.PresumedAbort);
+                            .Cancel(resource.Name, transactionId, timeStamp, TransactionalStatus.PresumedAbort).Ignore();
                 }
             }
         }
 
         public async Task NotifyOfConfirm(Guid transactionId, DateTime timeStamp)
         {
+            if (logger.IsEnabled(LogLevel.Trace))
+                logger.Trace($"NotifyOfConfirm: {transactionId} {timeStamp}");
```

Comment:
We should probably start using the "semantic logging" approach for new messages. That means removing the `$` and adding `transactionId` etc as parameters.

Not blocking.

Reply:
I'll do this in a separate targeted PR.  Don't want to clutter or delay this one, as it's blocking other changes.